README.md

```diff
@@ -16,68 +16,75 @@ To get started with the demo app, please ensure Flutter is installed on your sys
 4. Update the package name from `com.applovin.enterprise.apps.demoapp` to one that matches your ad units. Be sure to do this for every package name reference in the demo app. 
 
 ### Android
-#### 1. Adding Adapters to `build.gradle`:
+#### 1. Updating the package name in `app/build.gradle`:
 - Navigate to your Flutter project directory in your file explorer or terminal. 
 - Within the project directory, navigate to `android/app/` to find the `build.gradle` file. 
 - Open `build.gradle` with a text editor or an IDE. 
-- Add the adapters you need under dependencies, as the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#android) specifies. It will look something like this:
+- Update `applicationId` with your package name.
+````
+android {
+    ...
```

Comment:
Here and in line 29 below, [I recommend a vertical ellipsis (⋮) in cases when you are eliding a complete line or more than one complete lines of source code.](https://docs.google.com/document/d/1mvaE1khnBVsZA30Mau_S4bfjbqyB9uxoPXEDN45hopY/edit#heading=h.64o8utkic7ar)